import_data/p1.csv

```diff
@@ -0,0 +1,11 @@
+ID,EMAIL,PASSWORD,NAME,ROLE,MAJOR
+42267,duck@stevenscollege.edu,looney,Daffy Duck,teacher,CSET
+53114,doo@stevenscollege.edu,snacks,Scooby Doo,teacher,ARCH
+52502,mouse@stevenscollege.edu,disney,Minnie Mouse,teacher,CSET
+13041,bsimpson041@stevenscollege.edu,iwillnot,Bart Simpson,student,ARCH
+36706,lsimpson706@stevenscollege.edu,saxophone,Lisa Simpson,student,CSET
+70128,cbrown128@stevenscollege.edu,goodgrief,Charlie Brown,student,CSET
+25208,apickles208@stevenscollege.edu,spoiled,Angelica Pickles,student,CSET
+48175,bbunny175@stevenscollege.edu,carrots,Bugs Bunny,student,ARCH
+50425,fflintstone425@stevenscollege.edu,bowling,Fred Flintstone,student,ARCH
+82079,mmouse079@stevenscollege.edu,steamboat,Mickey Mouse,student,ARCH

```

Comment:
This file was already committed under a different name. Is there a reason to have two copies in here?